committing.rst

```diff
@@ -202,35 +202,19 @@ So a file name may be
 The contents of a news file should be valid reStructuredText. An 80 character
 column width should be used. There is no indentation or leading marker in the
 file (e.g. ``-``). There is also no need to start the entry with the issue
-number as it's part of the file name itself. Example news entry::
-
-   Fix warning message when ``os.chdir()`` fails inside
-   ``test.support.temp_cwd()``.  Patch by Chris Jerdonek.
-
-In addition to inline reST, news entries also support the usage of
-`Sphinx roles <https://devguide.python.org/documenting/#id4>`_. Several
-commonly used roles include :func:, :class:, and :meth:. When a
-corresponding entry is found within the documentation, the text within
-the role is converted into a hyperlink. Example news entry with Sphinx::
+number as it's part of the file name itself. You can use
+:ref:`inline markups <rest-inline-markup>` too. Example news entry::
 
    Fix warning message when :func:`os.chdir` fails inside
    :func:`test.support.temp_cwd`. Patch by Chris Jerdonek.
 
-The inline Sphinx roles can be used to assist readers in finding more
-information and context on the changes made. If the inline link does not
-provide additional context, an inline reST code block can be used instead::
-
-   ``<object>``
+The inline Sphinx roles like ``:func:`` can be used to assist readers in finding
+more information. You can build html and verify that the link target is
+appropriate by using :ref:`make html <building-using-make>`.
```

Comment:
Also, it looks like the [PR adding netlify support for cpython](https://github.com/python/cpython/pull/15288#issuecomment-523630256) was approved. Based on the latest comments, it looks like they're working on adding it as a GitHub app for the repository.

[As you had recommended](https://github.com/python/devguide/pull/525#discussion_r314977052), I think we should recommend for readers to use netlify since it's significantly more convenient and less time consuming than ``make html``. But, we could leave ``make html`` as an alternative.

Reply:
> Also, it looks like the [PR adding netlify support for cpython](https://github.com/python/cpython/pull/15288#issuecomment-523630256) was approved. Based on the latest comments, it looks like they're working on adding it as a GitHub app for the repository.

I know.  But it is not activated now so I can't document it for now.

Netlify preview will be activated and documented in "documenting" chapter.

If you want to wait it, wait merging this PR too.
If you don't want to wait it, I will merge this PR without referring about netlify.

Reply:
Based on Mariatta's last comment, it seems like we'll have to wait for the PSF to set up the account through netlify, so it might take some additional time. We can always update it at a later point in time.